neuralop/models/tfnogno.py

```diff
@@ -91,23 +112,18 @@ def __init__(
         )
         del self.fno.projection
```

Comment:
This feels weird to me

Reply:
What do you think of just using a Spectral Conv rather than a full FNO so we don't have to meddle with internals?

Reply:
Other option (which I think I like more) is to first provide a GNO layer out of the box and then just have a trivial concat of GNO and FNO?